essay_on_tigers.txt

```diff
@@ -0,0 +1,7 @@
+Tigers are magnificent creatures that belong to the cat family. They are known for their distinct orange fur with black stripes, which act as camouflage in their natural habitat. Tigers are the largest of all big cats and are renowned for their strength and agility.
+
+Tigers are solitary animals that prefer to live and hunt alone. They are highly territorial and mark their territory with scent markings and vocalizations. Tigers are carnivores and primarily feed on large ungulates such as deer and wild boar.
+
```

Comment:
Consider extracting the information about tigers being solitary animals into a separate paragraph.